src/test/java/core/basesyntax/strategy/filestrategy/DataParsingStrategyTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.strategy.filestrategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.exception.InvalidFileTypeException;
+import core.basesyntax.strategy.filestrategy.impl.CsvDataParserImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataParsingStrategyTest {
+    private static Map<String, DataParser> dataParsersMap;
```

Comment:
make it local, you use it only in beforeClass. Go Through all your test classes and make local variable from those fields, that used only in beforeClass methods